.github/workflows/ci-cd.yml

```diff
@@ -116,6 +116,47 @@ jobs:
         with:
           parallel-finished: true
 
+  package-test:
```

Comment:
Let's make the naming consistent: `build` + `test-build` or `package` + `test-package`